### PR TITLE
Added version information to libraries build with cmake

### PR DESCRIPTION
**Pull request by sgeto**
The whole nine yards. VERSION, SOVERSION and an embedded resource file for win32
screenshot 48

### CODE REVIEW

**Comment by guyharris** [comment]
You might as well just put `pcap-dll.rc` in the top-level directory; we don't have platform subdirectories for other OSes.

File: Win32/pcap-dll.rc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    BLOCK "StringFileInfo"
+    BEGIN
+      BLOCK "040904b0"
+      BEGIN
+        VALUE "Comments",         "https://github.com/the-tcpdump-group/libpcap/"
+        VALUE "CompanyName",      "The TCPdump Group"
+        VALUE "FileDescription",  "System-Independent Interface for User-Level Packet Capture"
+        VALUE "FileVersion",      "PACKAGE_VERSION_DLL"
+        VALUE "InternalName",     "PACKAGE_NAME"
+        VALUE "LegalCopyright",   "Copyright (c) The TCPdump Team"
```
Comment (guyharris) [inline Win32/pcap-dll.rc]: "TCPdump Group", not "TCPdump Team".

Comment (sgeto) [reply Win32/pcap-dll.rc]: Hehe I found that in README...

Comment (infrastation) [reply Win32/pcap-dll.rc]: The README now uses "group" consistently.

**Comment by guyharris** [comment]
> AppVeyor build failed

It failed to install WinFlexBison, which isn't your fault.

**Comment by sgeto** [comment]
pfew...

**Pull request merged**